### PR TITLE
Examples: make use of core utils in more places.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Comment by XhmikosR** [comment]
@twbs/css-review: If anyone wants to tackle the other examples, feel free to push on this branch.

File: site/content/docs/4.3/examples/sticky-footer-navbar/index.html [hunk 1]
```diff
@@ -38,13 +38,13 @@
 <main role="main" class="flex-shrink-0">
   <div class="container">
     <h1 class="mt-5">Sticky footer with fixed navbar</h1>
-    <p class="lead">Pin a footer to the bottom of the viewport in desktop browsers with this custom HTML and CSS. A fixed navbar has been added with <code>padding-top: 60px;</code> on the <code>main &gt; .container</code>.</p>
+    <p class="lead">Pin a footer to the bottom of the viewport in desktop browsers with this custom HTML and CSS. A fixed navbar has been added with <code class="small">padding-top: 60px;</code> on the <code>main &gt; .container</code>.</p>
```
Comment (ysds) [inline site/content/docs/4.3/examples/sticky-footer-navbar/index.html]:
```suggestion
    <p class="lead">Pin a footer to the bottom of the viewport in desktop browsers with this custom HTML and CSS. A fixed navbar has been added with <code class="small">padding-top: 60px;</code> on the <code class="small">main &gt; .container</code>.</p>
```

**Comment by XhmikosR** [comment]
Ping @twbs/css-review

**Comment by XhmikosR** [comment]
BTW, not sure if we should backport this to v4?

**Comment by mdo** [comment]
> BTW, not sure if we should backport this to v4?

Skip it

**Comment by XhmikosR** [comment]
Someone needs to approve it :P

**Review by mdo — APPROVED** [review]
Looking good, thanks! And to clarify, no need to backport :).

(Doing the same to the docs nav and more in some of my other branches fwiw. Plenty of stuff we can cleanup and dedupe.)

**Pull request merged**